### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by guohao**
Fix typo and refactor some code

### CODE REVIEW

File: boot/src/main/java/com/taobao/arthas/boot/Bootstrap.java [hunk 1]
```diff
@@ -219,8 +219,7 @@ public static void main(String[] args) throws ParserConfigurationException, SAXE
                 AnsiLog.info("arthas-boot version: " + arthasBootVersion);
             }
         }
-
-        String mavenMetaData = null;
+        String mavenMetaData=null;
```
Comment (hengyunabc) [inline boot/src/main/java/com/taobao/arthas/boot/Bootstrap.java]: 这里为啥去掉`=`左右的空格了？

Comment (guohao) [reply boot/src/main/java/com/taobao/arthas/boot/Bootstrap.java]: 我改一下

File: boot/src/main/java/com/taobao/arthas/boot/Bootstrap.java [hunk 2]
```diff
@@ -253,9 +252,7 @@ public static void main(String[] args) throws ParserConfigurationException, SAXE
         AnsiLog.debug("Repo mirror:" + bootstrap.getRepoMirror());
 
         if (bootstrap.isVersions()) {
-            if (mavenMetaData == null) {
-                mavenMetaData = DownloadUtils.readMavenMetaData(bootstrap.getRepoMirror(), bootstrap.isuseHttp());
-            }
+            mavenMetaData = DownloadUtils.readMavenMetaData(bootstrap.getRepoMirror(), bootstrap.isuseHttp());
```
[the remark this answers is not in the document]
Comment (guohao) [reply boot/src/main/java/com/taobao/arthas/boot/Bootstrap.java]: 好

**Comment by guohao** [comment]
@hengyunabc 再看下哈

**Comment by hengyunabc** [comment]
Thank you very much.

**Pull request merged**